DOCS-10828 teams (June 12) (#136)
mulesoft/docs-exchange: modules/ROOT/pages/about-sharing-assets.adoc

= Publish Assets
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> Publish APIs, API Groups, templates, connectors, policies, Mule XML example code, and custom information assets to a private Exchange or an Exchange public portal. Exchange provides a repository for code and APIs, and helps your organization access them easily, model them, and reuse them.

Reviewer: ```suggestion
Publish APIs, API Groups, templates, connectors, policies, Mule XML example code, and custom information assets to a private Exchange server or an Exchange public portal. Exchange provides a repository for code and APIs, and helps your organization access them easily, model them, and reuse them.
```

Author: Fixed.

> A private Exchange makes it easy to provide assets to internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically provided to the developers of that business group.

Reviewer: ```suggestion
A private Exchange server makes it easy to provide assets to internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically provided to the developers of that business group.
```
Are we distinguishing between role names and access types? They are init. capped earlier, lowercased here.

Author: Yes, the role names are Exchange Viewer, Exchange Contributor, Exchange Administrator, Asset Viewer, Asset Contributor, and Asset Administrator, but here we're just describing the asset types, so I'm using lower case.

> An Exchange public portal enables Exchange administrators to provide assets externally to developers outside of their own organization.

Reviewer: ```suggestion
An Exchange public portal enables Exchange administrators to provide assets to external developers.
```
OR
```suggestion
An Exchange public portal enables Exchange administrators to provide assets to developers outside of their own organizations.
```

Author: Fixed.
